HotelBooking/HotelBooking.App/Operations/ExternalSystems/IExternalEmailSystem.cs

```diff
@@ -0,0 +1,10 @@
+﻿using HotelBooking.Domain.Models;
+
+namespace HotelBooking.App.Operations.ExternalSystems
+{
+    public interface IExternalEmailSystem

```

Comment:
To jest wlasnie przyklad chyba nie dokonca dobrej nazwy w tej aplikacji. External nie powinien byc tu podmiotem. To jest tylko szczegol. Rownie dobrze moze byc Internal. Dlatego nazwalbym to poprostu IEmailSystem.